bin/exec.go

```diff
@@ -85,93 +77,49 @@ func main() {
 
 		// If not found in AppNamespace skip the further steps
 		if !isFound {
+			//TODO Patch Engine if chaosExperiment is not found.
+			engineDetails.ExperimentNotFoundPatchEngine(experiment, clients)
+
 			log.Infoln("Can't Find Experiment Name : "+engineDetails.Experiments[i], "In Namespace : "+engineDetails.AppNamespace)
 			log.Infoln("Not Executing the Experiment : " + engineDetails.Experiments[i])
 			break
 		}
+		log.Info("Printing the ENV Variables", experiment.Env)
```

Comment:
Hide this comment. Or print a structured log as a key:value

Reply:
Done, removed the log.